### PR TITLE
fix:  chart label formatting and pagination strategy

### PR DESCRIPTION
**Pull request by pocin**
# Description
What does this PR solve?

Fixes # (issue)

# Checklist
- [ ] Code follows the style guide
- [ ] I have performed a self-review of my own code
- [ ] New and existing tests pass locally
- [ ] This PR is targeting the correct branch

### CODE REVIEW

**Comment by nichosystem** [comment]
Deploy preview for _core_ ready!

✅ Preview
https://core-qnymvjpp8-templedao.vercel.app
https://pr-979-preview.templedao.link

Built with commit 624e6cf42f0880503fe58bd46d91025a54330f06.
This pull request is being automatically deployed with [vercel-action](https://github.com/marketplace/actions/vercel-action)

File: apps/dapp/src/components/Pages/Core/DappPages/Dashboard/Chart/V2StrategyMetricsChart.tsx [hunk 1]
```diff
@@ -73,17 +73,21 @@ const V2StrategyMetricsChart: React.FC<{
   selectedInterval: ChartSupportedTimeInterval;
 }> = ({ dashboardType, selectedMetric, selectedInterval, strategyNames }) => {
   // uncamel-case the metric names
-  const formatMetricName = (name: string) =>
+  const formatMetricName = (name: string) => {
+
     // format only the selected metric name (the selected metric or all lines in a TRV chart)
-    name === selectedMetric || dashboardType === DashboardType.TREASURY_RESERVES_VAULT
-      ? name
+    if (name === 'debtUSD') return 'Debt (USD)';
```
Comment (frontier159) [inline apps/dapp/src/components/Pages/Core/DappPages/Dashboard/Chart/V2StrategyMetricsChart.tsx:79]: A bit gross why does it come through like this?

Comment (frontier159) [reply apps/dapp/src/components/Pages/Core/DappPages/Dashboard/Chart/V2StrategyMetricsChart.tsx:79]: I haven't dug into this code, but is it possible to pass through a formatter lambda at runtime which says how to format that particular graph?

Comment (frontier159) [reply apps/dapp/src/components/Pages/Core/DappPages/Dashboard/Chart/V2StrategyMetricsChart.tsx:79]: Can approve for now, but yeh it's not very scalable

**Review by frontier159 — DISMISSED** [review]

**Review by medariox — DISMISSED** [review]
👍

**Review by medariox — APPROVED** [review]

**Pull request merged**